inst/tools/workaround.R

```diff
@@ -26,7 +27,7 @@ for (f in c("inst/include/RxODE_RcppExports.h", "src/RcppExports.cpp")) {
                                .badStan)),
             .in)
 
-.in <- gsub("@SL@", "", ##paste(capture.output(StanHeaders:::LdFlags()), capture.output(RcppParallel:::RcppParallelLibs())),
```

Comment:
@andrjohns 

Maybe try commenting this out again.  The main branch seems to be working OK.